l10n_it_vat_registries_split_payment/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+**Italiano**
+
+Modulo di congiunzione tra le funzionalità legate ai registri IVA e le
+funzionalità inerenti le scissione dei pagamenti.
```

Comment:
> @eLBati e @primes2h; ora dovrebbe essere a posto! joy

Quasi, c'è ancora un typo da correggere. :smiley_cat: 

s/le/la

Reply:
> s/le/la

Non sono stato in grado di identificare a cosa facesse riferimento la tua prima "s" (forse voleva indicare una "**S**ostituzione"?).  

In ogni caso, ho corretto come segue:
```suggestion
funzionalità inerenti alla scissione dei pagamenti.
```

Reply:
> > s/le/la
> 
> Non sono stato in grado di identificare a cosa facesse riferimento la tua prima "s" (forse voleva indicare una "**S**ostituzione"?).

Si, è una convenzione usata nella revisione di documentazione/traduzioni, penso derivi da `sed`.

s/`<stringa da sostituire>`/`<stringa suggerita>` (sostituzione)
d/`<stringa da rimuovere>` (rimozione)

> In ogni caso, ho corretto come segue:

Si, più corretto. :+1: